crates/runc-rust/Cargo.toml

```diff
@@ -0,0 +1,28 @@
+[package]
+name = "containerd-runc-rust"
```

Comment:
runc is independent from containerd.
can we name this just `runc` ?

Reply:
sounds good, thank you. 
And I'll split PR.